### ISSUE TITLE
TIMESTAMP value anomaly

### ISSUE THREAD

**Issue by VWagen1989** [apecloud/myduckserver#225]
In the test of `TestBinlogReplicationForAllTypes`, the case failed because TIMESTAMP value unmatched:
![image](https://github.com/user-attachments/assets/ff9191a4-2bed-4e95-8867-0a9605552031)
Maybe this is due to the unmatch of time zone between primary MySQL(In UTC) and the replica MyDuck Server(In +08:00).

**Comment by fanyang01** [comment]
I have never observed this failure. How was it produced? Are your running MyDuck Server in non-default `TimeZone` setting?

**Comment by VWagen1989** [comment]
> I have never observed this failure. How was it produced? Are you running MyDuck Server in non-default `TimeZone` setting?

No. The MyDuck Server is launched by the test case and I think maybe it use the local time zone on my Mac which is Asia/Shanghai, aka +08:00. But the source data is from the MySQL instance which is lauched in a docker container and the time zone is UTC, aka +00:00. So the time value on MyDuck Server is 8 hours ahead comparing to the one in MySQL.
